### PR TITLE
Minor grammar and clarity modifications to amazon-redshift.mdx

### PR DESCRIPTION
**Pull request by marcoscandroglio**
## Description

Provides some minor grammar and clarity modifications to the Amazon Redshift documentation

Fixes #issue_number

## Type of change

- [ ] 📄 This change requires a documentation update

### CODE REVIEW

**Comment by ZoranPandovski** [comment]
recheckcla

**Comment by mindsdbadmin** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/mindsdb/mindsdb/blob/staging/assets/contributions-agreement/individual-contributor.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>
   - - -
   I have read the CLA Document and I hereby sign the CLA
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

File: docs/integrations/data-integrations/amazon-redshift.mdx [hunk 1]
```diff
@@ -5,11 +5,11 @@ sidebarTitle: Amazon Redshift
 
 This is the implementation of the Redshift data handler for MindsDB.
 
-[Amazon Redshift](https://docs.aws.amazon.com/redshift/latest/mgmt/welcome.html) is a fully managed, petabyte-scale data warehouse service in the cloud. You can start with just a few hundred gigabytes of data and scale to a petabyte or more. This enables you to use your data to acquire new insights for your business and customers.
+[Amazon Redshift](https://docs.aws.amazon.com/redshift/latest/mgmt/welcome.html) is a fully managed, petabyte-scale data warehouse service in the cloud. You can start with just a few hundred gigabytes of data and scale to a petabyte or more enabling you to use your data to acquire new insights for your business and customers.
```
Comment (martyna-mindsdb) [inline docs/integrations/data-integrations/amazon-redshift.mdx:8]: There should be a comma here: `...to a petabyte or more, enabling you to use...` to make it correct.

Comment (marcoscandroglio) [reply docs/integrations/data-integrations/amazon-redshift.mdx]: @martyna-mindsdb , thank you for catching that.  The missing comma has been added.

**Comment by marcoscandroglio** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by ZoranPandovski — APPROVED** [review]

**Pull request merged**